lib/Backend/GlobOpt.cpp

```diff
@@ -1060,6 +1060,9 @@ GlobOpt::MergePredBlocksValueMaps(BasicBlock *block)
         this->KillAllObjectTypes();
     }
 
+    this->blockData.capturedArgs = JitAnew(this->alloc, BVSparse<JitArenaAllocator>, this->alloc);
+    this->blockData.changedSyms = JitAnew(this->alloc, BVSparse<JitArenaAllocator>, this->alloc);
```

Comment:
Yes, reuse is good. Will take this change in the cross block version.

---
In reply to: [68328456](https://github.com/Microsoft/ChakraCore/pull/1152#discussion_r68328456) [](ancestors = 68328456)